models/vrp.rb

```diff
@@ -456,6 +465,12 @@ def self.detect_date_indices_inconsistency(hash)
     def self.generate_schedule_indices_from_date(hash)
       return hash if !hash[:configuration] || !hash[:configuration][:schedule]
 
+      schedule = hash[:configuration][:schedule]
+      if (!schedule[:range_indices] || [schedule[:range_indices][:start], schedule[:range_indices][:end]].compact.size < 2) ||
+         (!schedule[:range_date] || [schedule[:range_date][:start], schedule[:range_date][:end]].compact.size < 2)
```

Comment:
This condition is wrong. It raises if we have proper `schedule[:range_indices]` with `start` and `end` but no `schedule[:range_date]`, no?

I also suggest replacing `[].compact.size < 2` with `[].any?(&:nil?)` or just good old `schedule[:range_indices][:start].nil? || schedule[:range_indices][:end].nil?` to be more expressive -- i.e., "if any of them nil?" instead of "if the size of the array without the nils is less than two?"